docs/source/contributing/developing.adoc

```diff
@@ -280,6 +280,17 @@ For example, tests can be run against a stopped {project} instance with the imag
 $ make integration_all RUN_BEFORE_FEATURE="start; stop; config set image-caching true"
 ----
 
+COPY_OC_FROM::
```

Comment:
Hi @robin-owen, could I kindly ask you to take a look on this paragraph? Would like to check whether it is understandable and correct, thanks :)